src/cdk/utils/create-unauthorized-gateway-response.ts

```diff
@@ -7,30 +7,31 @@ export function createUnauthorizedGatewayResponse(
   stack: Stack,
   restApi: RestApi
 ): void {
-  const {basicAuthentication} = stackConfig;
+  const {basicAuthenticationConfig: basicAuthentication} = stackConfig;
```

Comment:
Why the renaming? I think we can use the property directly without destructuring

Reply:
accidental via F2